workshops/views.py

```diff
@@ -33,7 +33,8 @@
 from workshops.check import check_file
 from workshops.forms import (
     SearchForm, DebriefForm, InstructorsForm, PersonBulkAddForm, EventForm,
-    TaskForm, bootstrap_helper, bootstrap_helper_without_form)
+    TaskForm, bootstrap_helper, bootstrap_helper_without_form,
+    PersonForm, TaskFullForm)
```

Comment:
Very minor thing: you can put these imports to order: first forms, then helpers.